opentelemetry-sdk/src/opentelemetry/sdk/environment_variables.py

```diff
@@ -635,3 +635,14 @@
 
 The :envvar:`OTEL_EXPORTER_OTLP_METRICS_CLIENT_CERTIFICATE` is the client certificate/chain trust for clients private key to use in mTLS communication in PEM format.
 """
+
+OTEL_EXPERIMENTAL_RESOURCE_DETECTORS = "OTEL_EXPERIMENTALRESOURCE_DETECTORS"
+"""
+.. envvar:: OTEL_EXPERIMENTAL_RESOURCE_DETECTORS
+
+The :envvar:`OTEL_EXPERIMENTAL_RESOURCE_DETECTORS` is a comma-separated string
+of names of resource detectors. These names must be the same as the names of
+entry points for the `opentelemetry-resource-detectors` entry point. This is an
```

Comment:
```suggestion
entry points for the `opentelemetry_resource_detector` entry point. This is an
```

Reply:
I believe the entry point group name must match the actual implementation name in the documentation.